dell/xps/15-9500/default.nix

```diff
@@ -0,0 +1,10 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ./xps-common.nix
+  ];
+
+  # This configuration makes intel default
+  services.xserver.videoDrivers = lib.mkDefault [ "intel" ];
```

Comment:
The default setting here is: `[ "radeon" "cirrus" "vesa" "modesetting" ]`.
Modesetting is the preferred driver for intel graphic cards according to my information.

```suggestion
```